packages/components/src/molecules/ReviewCard/ReviewCardDate.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+import { formatDate } from '../../utils/date'
+
+export interface ReviewCardDateProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The date of the review.
+   */
+  date: Date
+  /**
+   * Optional locale to format the date. Defaults to 'en-US'.
+   */
+  locale?: string
+  /**
+   * The string label to display when the date is today.
+   */
+  todayLabel: string
+}
+
+const ReviewCardDate = forwardRef<HTMLDivElement, ReviewCardDateProps>(
+  function ReviewCardDate(
+    { date, locale = 'en-US', todayLabel = 'Today', ...otherProps },
```

Comment:
Good 💯

Reply:
The `todayLabel` should come from CMS.

Reply:
Niceee! We can to this CMS input change later. But if so, let's create a task 🙇